docs/src/guides/typo3/deploy/customize.md

```diff
@@ -0,0 +1,117 @@
+
+```bash
+$ composer config extra.typo3/cms.web-dir public && composer update --no-scripts
+```
+
+### Platform.sh Configuration Reader library for PHP
+
+TYPO3, like all applications on Platform.sh, will read information about it's environment through a collection of pre-defined environment variables. Some of them are straight forward and highly visible (i.e. `$PLATFORM_BRANCH=platformify`), while others are base64 encoded JSON objects that contain nested detailed information about that environment (i.e. `$PLATFORM_RELATIONSHIPS`).
+
+In order to make these variables, and any other you set in the future, easier to read and use within TYPO3 it's recommended that you install Platform.sh's [Configuration Reader library for PHP](https://github.com/platformsh/config-reader-php). It contains a set of common functions for interacting with the environment within a PHP application. 
+
+```bash
+$ composer require platformsh/config-reader
+```
+
+### Avoiding deadlock with the Local Page Error Handler
+
+A default TYPO3 installation has a risk of deadlocks when run on low-worker PHP-FPM configurations. Specifically, TYPO3 handles 403 and 404 error pages by issuing a full HTTP request back to itself with no timeout, which can lead to process starvation. There are two required steps to avoid this problem.
+
+First, you will need to install the [Local Page Error Handler](https://extensions.typo3.org/extension/pxa_lpeh/) extension for TYPO with the command:
+
+```bash
+$ composer require pixelant/pxa-lpeh
+```
+
+Second, you will need to add a timeout which will set an HTTP timeout of 3 seconds instead of the default several minutes to a new `public/typo3conf/PlatformshConfiguration.php`. You can see this line in context of the full file in the [configuration](#environment) section below.
+
+```php
+$GLOBALS['TYPO3_CONF_VARS']['HTTP']['timeout'] = 3;
+```
+
+You will still need to enable the `pixelant/pxa-lpeh` extension, which you will be able to do in the [post-install](/guides/typo3/deploy/deploy.md#post-install) steps later in this guide.
+
+### (Optional) Introductory examples
+
+The TYPO3 template for Platform.sh loads a number of example content pages provided by TYPO3 into the site when dependencies are installed pre-build. Their inclusion is completely optional, but a helpful choice if you are creating a new TYPO3 site from scratch using Composer.
+
+```bash
+$ composer require typo3/cms-introduction
+```
+
+## Configuration
+
+You're going to add and/or edit a few files described in the sections below, which will result in the following directory structure:
+
+```bash
+.
+├── config
+│   └── sites
+│       └── main
+│           └── config.yaml
+...
+├── public
+│   └── typo3conf
+│       ├── AdditionalConfiguration.php
+│       └── PlatformshConfiguration.php
+└── src
+    ├── SetupConfiguration.yaml
+    └── SetupDatabase.yaml
```

Comment:
If you're starting with an existing site, it's not so likely that you have these files. They're not a standard part of TYPO3.